packages/afl/afl.2.57b/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+maintainer: "meetup@yomimono.org"
+homepage: "https://github.com/google/AFL"
+dev-repo: "git+https://github.com/google/AFL.git"
+bug-reports: "https://groups.google.com/forum/#!forum/afl-users"
+license: "Apache-2.0"
+build: [
+  [make] {arch = "x86_64"}
+  [make "AFL_NO_X86=1"] {arch != "x86_64"}
+]
+install: [
+  [make "PREFIX=%{prefix}%" "install"] {arch = "x86_64"}
+  [make "AFL_NO_X86=1" "PREFIX=%{prefix}%" "install"] {arch != "x86_64"}
+]
+remove: [
+  [make "PREFIX=%{prefix}%" "uninstall"]
+]
+patches: [
+  "add-uninstall-target.patch"
+]
+synopsis:
+    "American Fuzzy Lop fuzzer by Michal Zalewski, repackaged for convenient use in opam"
+authors: "Michal Zalewski"
+depends: ["ocaml"]
```

Comment:
I don't understand this, the package only depends on a c compiler, no?

Reply:
Woops, I merged before seeing this. I think @mseri is right...

Reply:
Yes it only depends on a c compiler. 
What's the correct way to depend on that? @mseri

Reply:
I think it is `(conf-gcc | conf-clang)` but I am not 100% sure. If there is ocaml, most likely there is also the c compiler, so even though it is not strictly required perhaps is a good idea to keep it as is